src/styles/_variables.scss

```diff
@@ -22,7 +22,7 @@ $md-gray-95: #1C1C1C !default;
 $md-gray-100: #121212 !default;
 $md-yellow-40: #FC9D03 !default;
 
-$md-red-60: #D4371C !default; 
+$md-red-60: #D4371C !default;
```

Comment:
if you didn't do any other modification to this file, let's leave it like it was, these colors will be all removed anyway

Reply:
not resolved

Reply:
still not resolved. Use another editor if possible. You have to add a blank space after the semicolon to revert this change.

Reply:
still not resolved